### PR TITLE
Display "No Class" after school's out

### LINKED ISSUE #8: "Your next class period is" displays current day's first period instead of next day's first period
**Describe the bug**
"Your next class period is:" displays current day's first period instead of next day's first period

**To Reproduce**
1. Go to the homepage on finals schedule.
2. Look at the "your next class period" section.

**Expected behavior**
(On Wed, Jan 23rd)
Your next class period is A3 Final.

### PR DESCRIPTION
**Pull request by nbd9**
Fixes #8

### CODE REVIEW

**Comment by MoralCode** [comment]
So before you made this change it said "1st period" under "next class" both before and after school,
after this change, it says "no class" as the next class before school has started. This is because a `currentClassPeriodIndex` of `-1` just means "there is currently no class in session" without differentiating before vs after school. (so when you pass `-1` into `getClassName()` it returns "No Class")


TL;DR you'd need to add a check for before and after school

**Comment by MoralCode** [comment]
https://github.com/MoralCode/ClassClock/wiki/Time-States

**Comment by MoralCode** [comment]
There are also some issues with merging this to dev branch. The app is mis-handling the gaps in the schedule that are meant to be passing periods and so it says you are in no class and your next class is 1st period no matter what passing period you are in. i'll try to fix this through improving the way the app determines what class period it is
